nncf/onnx/graph/metatypes/groups.py

```diff
@@ -121,3 +121,9 @@
     onnx_metatypes.ONNXConvolutionMetatype,
     onnx_metatypes.ONNXDepthwiseConvolutionMetatype,
 ]
+
+# These metatypes lead to inaccurate statistics when batch size > 1.
```

Comment:
We do not do batch searching in general. Output of These metatypes has no batch axis anymore. I could add these

Reply:
Please explain in the comment what are the reasons behind accuracy degradation. Accuracy degradation is consequence of some kind of error

Reply:
In case batch size > 1 the tensor could mix outputs for different samples into one axis and we simply reduce them and get 1 value instead batch_size values. This leads to inaccurate statistics

Reply:
I can extend this comment by the description above.

Reply:
That would be great!

Reply:
done